pages/api-reference/customers/create.mdx

```diff
@@ -7,8 +7,6 @@ meta:
   title: "Create a Customer | Dwolla API Documentation"
 ---
 
-import guideIcon from "../../../assets/images/content-images/content-icons/guides-icon.svg";
```

Comment:
Removed this import since `guideIcon` is not used anywhere else on this page. Happy to add it back, though, if requested.